projects/Mallard/src/hooks/use-cached.ts

```diff
@@ -0,0 +1,63 @@
+import NetInfo from '@react-native-community/netinfo'
+
+import { useEffect, useState } from 'react'
+import { useSettings } from './use-settings'
+import axios from 'axios'
+import RNFetchBlob from 'rn-fetch-blob'
+import { DatePickerAndroid } from 'react-native'
+
+export const useCached = <Fetchable>(
```

Comment:
Are we going with non-descriptive type names for generic types without a description?

Reply:
Yeah, I'd just been copy and pasting